master/internal/task/allocation.go

```diff
@@ -317,6 +317,18 @@ func (a *allocation) SetProxyAddress(_ context.Context, address string) error {
 	return nil
 }
 
+// SetAcceleratorData adds the accelerator data of the allocation.
+func (a *allocation) SetAcceleratorData(ctx context.Context, containerID string,
+	nodeName string, acceleratorType string, acceleratorUuids []string,
+) error {
+	if err := db.AddAllocationAcceleratorData(ctx, containerID, a.model, nodeName, acceleratorType,
+		acceleratorUuids); err != nil {
+		a.crash(err)
```

Comment:
do you have any strong opinions on crashing the allocation verse just logging an error?

Reply:
No strong opinions. Its not that severe of an error to warrant crashing